osquery/core/system.cpp

```diff
@@ -149,6 +148,7 @@ std::string generateHostUUID() {
   }
 
   // Unable to get the hardware UUID, just return a new UUID
+  VLOG(1) << "Cannot retrieve platform UUID: generating an ephemeral UUID";
```

Comment:
@theopolis is there any issue with me moving this verbosity up one level? I figure we don't need this for just generating a new `UUID`, and the other functions which leverage `generateNewUUID` all have their own verbosity. Thoughts?